test/e2e/framework/pod/resource.go

```diff
@@ -379,7 +378,8 @@ func FilterNonRestartablePods(pods []*v1.Pod) []*v1.Pod {
 }
 
 func isNotRestartAlwaysMirrorPod(p *v1.Pod) bool {
-	if !kubetypes.IsMirrorPod(p) {
+	_, ok := p.Annotations[v1.MirrorPodAnnotationKey]
```

Comment:
Could you add a comment here, please?
Something simple like "checking that pod is mirror pod" or something.

Reply:
`if IsMirrorPod(p)` is kind of self-descriptive.
`if _, ok := p.Annotations[v1.MirrorPodAnnotationKey]; !ok` would do well with some additional context,I think.

Reply:
sure , i will update it. it looks good with comment.